models/org_team.go

```diff
@@ -194,9 +194,25 @@ func (t *Team) RemoveRepository(repoID int64) error {
 	return sess.Commit()
 }
 
+func IsUsableTeamName(name string) (err error) {
+	var reservedTeamNames = []string{"new"}
+
+	for i := range reservedTeamNames {
+		if name == reservedTeamNames[i] {
```

Comment:
Macaron isn't (apparently) case insensitive ( test by going to `/org/CREATE` instead of `/org/create` )